### PR TITLE
get a prod package

### PR DESCRIPTION
**Pull request by monsieurleberre**

### CODE REVIEW

File: .github/workflows/delete.packages.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  
+  delete:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Delete packages
+      id: deletePackages
+      uses: actions/delete-package-versions@v3
```
Comment (joaofbvieira) [inline .github/workflows/delete.packages.yml]: Not a blocker for this PR, but could use `actions/delete-package-versions@v4` instead. There is an example in common repository.

File: src/Common.Projects.props [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project>
     <PropertyGroup>
         <Company>Trakx</Company>
-        <Copyright>Copyright © 2021 Trakx</Copyright>
+        <Copyright>Copyright © 2022 Trakx</Copyright>
```
Comment (joaofbvieira) [inline src/Common.Projects.props]: Not a blocker for now, but now it is 2023.

**Review by joaofbvieira — DISMISSED** [review]
LGTM. Added some suggestion, but are not a blocker for this PR.
Maybe need to solve merge conflicts before merge the PR.

File: src/Common.Projects.props (hunk 2 above)
Comment (monsieurleberre) [reply src/Common.Projects.props]: thanks I opened #113 for it :)

File: .github/workflows/delete.packages.yml (hunk 1 above)
Comment (monsieurleberre) [reply .github/workflows/delete.packages.yml]: thanks, dependabot's PR just merged 🥳

Comment (joaofbvieira) [reply .github/workflows/delete.packages.yml]: @monsieurleberre in order to use the new version of "delete-package-versions", it is required to add an extra parameter `package-type: 'nuget'`. This was already done in [common](https://github.com/trakx/common) repository.
image

Comment (monsieurleberre) [reply .github/workflows/delete.packages.yml]: OK Sorry I didn't see that, will do the change now! 🙏

Comment (monsieurleberre) [reply .github/workflows/delete.packages.yml]: https://github.com/trakx/shrimpy-api-client/pull/114

**Review by joaofbvieira — APPROVED** [review]
LGTM xD

**Pull request merged**